srcpkgs/wezterm/template

```diff
@@ -0,0 +1,56 @@
+# Template file for 'wezterm'
+pkgname=wezterm
+version=20220905
+revision=1
+_srcver=20220905-102802-7d4b8249
+wrksrc="${pkgname}-${_srcver}"
+build_style=cargo
```

Comment:
```suggestion
build_style=cargo
configure_args="--features distro-defaults"
```
https://github.com/wez/wezterm/blob/main/README-DISTRO-MAINTAINER.md#building-wezterm

would also be good to not use the bundled fonts

Reply:
i don't think you need to patch the fonts out, there are features flags for it

Reply:
I've applied the items mentioned in the document.

A [recent commit](https://github.com/wez/wezterm/commit/ad35b9dc217c9223aa93d963707e70233690d573) (22/9/9) discusses these feature flags, they do not exist as shipped on 22/9/5. In lieu of that, I added the two font packages we have and disabled binary inclusion.

-Joe

Reply:
ah i see